third_party/move/tools/move-formatter/tests/formatter/tuple/input4.move.fmt

```diff
@@ -0,0 +1,40 @@
+address 0x42 {
+    module example {
+        fun examples(cond: bool) {
+            // This line is an example of a unit value being assigned to a variable  
+            let () = ();
+            
+            // This line is an example of a tuple with multiple types being assigned to variables a, b, c, and d  
+            let (a, b, c, d) = (@0x0, 0, false, b"");
+            
+            // Reassignment of unit value  
+            () = ();
+            
+            // Conditional reassignment of tuple values x and y  
+            (x, y) = if (cond)(1, 2) else (3, 4);
```

Comment:
```suggestion
            (x, y) = if (cond) (1, 2) else (3, 4);
```

Reply:
Seems weird to have removed the space there.

Reply:
Updated.